addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -464,10 +464,8 @@ odoo.define('website.s_website_form', function (require) {
                     $field.addClass('o_has_error');
                     $controls.addClass('is-invalid');
                     if (_.isString(error_fields[field_name])) {
-                        $field.popover({content: error_fields[field_name], trigger: 'hover', container: 'body', placement: 'top'});
-                        // update error message and show it.
-                        $field.data("bs.popover").config.content = error_fields[field_name];
-                        $field.popover('show');
+                        let popover = new Popover($field, {content: error_fields[field_name], trigger: 'hover', container: 'body', placement: 'top'});
+                        popover.show();
```

Comment:
@pimodoo cc @rfr-odoo 

Seems good to me but:
- Use `const` instead of `let` (detail)
- How can I reproduce the issue here? (maybe you can add it in the commit message? Any task ID maybe too?) :pray: 
- I think we already had to fix some issues like that before... so this time I grepped `.popover(` in our codebase, there are 100 occurrences (grepping `.popover({`, there are 25, maybe it's only the initialization that does not work like that anymore?) :| How do they work? I tested one (the shop cart), it works :shrug: Maybe @rfr-odoo can give insights here?

Reply:
Bootstrap 5 has a compatibility layer for the use of jQuery, so it should works.
I think the only change is how access to the instance of the Popover.
```javascript
$field.data("bs.popover"); // don't return the instance of the Popover anymore
```
We should use this instead:
```javascript
  const popover = bootstrap.Popover.getInstance($field);
  popover._config.content = error_fields[field_name];
```

Reply:
it can be reproduced here, by validating the form without all required fields: https://www.odoo.com/fr_FR/accounting-firms/register (in the console without debug or in a popup in debug mode)